internal/ir/xds.go

```diff
@@ -1478,11 +1478,12 @@ type JSONAccessLog struct {
 // OpenTelemetryAccessLog holds the configuration for OpenTelemetry access logging.
 // +k8s:deepcopy-gen=true
 type OpenTelemetryAccessLog struct {
-	Text       *string           `json:"text,omitempty" yaml:"text,omitempty"`
-	Attributes map[string]string `json:"attributes,omitempty" yaml:"attributes,omitempty"`
-	Host       string            `json:"host" yaml:"host"`
-	Port       uint32            `json:"port" yaml:"port"`
-	Resources  map[string]string `json:"resources,omitempty" yaml:"resources,omitempty"`
+	Text         *string               `json:"text,omitempty" yaml:"text,omitempty"`
+	Attributes   map[string]string     `json:"attributes,omitempty" yaml:"attributes,omitempty"`
+	Host         string                `json:"host" yaml:"host"`
+	Port         uint32                `json:"port" yaml:"port"`
+	Resources    map[string]string     `json:"resources,omitempty" yaml:"resources,omitempty"`
+	Destinations []*DestinationSetting `json:"destinations,omitempty" yaml:"destinations,omitempty"`
```

Comment:
probably want to use `RouteDestination` here
https://github.com/envoyproxy/gateway/blob/c5ec046be14c4bb9049a7d3633900d2b8207d40f/internal/ir/xds.go#L911 to reuse clusters internally

Reply:
let's do that in 1.2, in which EG remove support for host and port?

Reply:
imo its a small change we can handle here

Reply:
reusing clusters is already handled in xds, so we only need to populate the new struct and read from it

Reply:
will take a look later